app/client/cli/doc/CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fixed message signing
 - Reporting RPC StatusCode and body
+- System commands working end-to-end
+- Added Consensus RoundState commands
```

Comment:
Let's just call them `State` commands, and `RoundState` is an implied "subcommand". The terminology is more future-proof that way.